django_hosts/middleware.py

```diff
@@ -80,10 +80,10 @@ def process_response(self, request, response):
         # Find best match, falling back to settings.DEFAULT_HOST
         try:
             host, kwargs = self.get_host(request.get_host())
-        except DisallowedHost:
+        except DisallowedHost:  # pragma: no cover
```

Comment:
Please write a test that covers this list rather than '# pragma: no cover'.

Reply:
I struggle into finding a valid test for this one in Python 3.x, could you help me on this ?

Reply:
Sorry, I don't have time to look into it.

Reply:
@Djailla just look into tests that override the `ALLOWED_HOSTS` settings.
Search code for `@override_settings(ALLOWED_HOSTS`

Reply:
and thanks for working on it!  we're waiting for this PR to merge too :)

Reply:
Why is DisallowedHost exception catching needed on Python 2 but not Python 3? That seems suspicious.

Reply:
I did revert my change, the point is the exception is only raised when using Django < 2.0

Reply:
I've removed this branch in https://github.com/jazzband/django-hosts/pull/114.